dowhy/causal_refuters/placebo_treatment_refuter.py

```diff
@@ -40,6 +42,8 @@ def __init__(self, *args, **kwargs):
             self._placebo_type = "Random Data"
         self._num_simulations = kwargs.pop("num_simulations",CausalRefuter.DEFAULT_NUM_SIMULATIONS)
         self._random_state = kwargs.pop("random_state",None)
+        self._n_jobs = kwargs.pop('n_jobs', None) # None or 1 as default?
```

Comment:
Should these be method specific parameters or added to `CausalModel.refute_estimate`? Especially if all methods can do parallelization.

Reply:
better to keep it method-specific. 

None as default is good. 
verbose is useful too.

Reply:
Are you sure about `n_jobs=None` as the default because it's slightly different to `n_jobs=1`, see docs here https://joblib.readthedocs.io/en/latest/generated/joblib.Parallel.html

Reply:
If I understood correctly, it default to n_jobs=1 (sequential), but still allows advanced users to set a global ParallelContext, right? That felt like a good balance to me.

Reply:
Yeah that makes sense!!